parsl/tests/test_manual/test_regression_220.py

```diff
@@ -0,0 +1,28 @@
+from parsl import DataFlowKernel, set_stream_logger
```

Comment:
I do not understand this test. Is this meant to be run automatically?

Reply:
No. You need to edit `/etc/resolv.conf` and add a bad nameserver line `nameserver 2.2.2.2` so as to break the networking. Only then can you run this test.